src/styles/blocks/shop.scss

```diff
@@ -0,0 +1,57 @@
+.shop {
+
+  &__title {
+    @extend %text-h2;
+    text-transform: uppercase;
+    margin-bottom: 20px;
+
+    @include Tablet {
+      margin-bottom: 15px;
+    }
+
+    @include Desktop {
+      margin-bottom: 30px;
+    }
+  }
+  
+  &__nav {
+
+    &--list {
+      display: flex;
+      flex-direction: row;
+      justify-content: space-between;
+      list-style: none;
+      gap: 20px;
+      font-size: 14px;
+      letter-spacing: 5%;
+      font-weight: 500;
+      text-transform: uppercase;
+      margin-bottom: 25px;
+
+      @include Tablet {
+        margin-bottom: 30px;
+      }
+
+      @include Desktop {
+        margin-bottom: 60px;
+        font-size: 18px;
+      }
+    }
+
+    &--link {
```

Comment:
Use a more descriptive variable name for the color, like '$link-color', instead of '$border-links-color'.